### DISCUSSION TITLE
水桶/水单元与流体/固体装罐器bug，部分机器无法合成，弹出升级没有功能

### DISCUSSION THREAD

**Discussion by Stephen633** [IndustrialCraftMC/IndustrialCraft2#119] [【漏洞反馈】Bugs Report]
forge是1.19.2的2.23 forge，ic2是 ic2-forge-unspecified+551d017，其他mod如下图所示
![ic2测试](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/72491010/399b9004-e2c6-41bc-990a-aa4b1fe10796)

我最近测试沼气机，由于流体弹出升级暂时没有功能，所以直接用水桶和水单元测试，使用水桶shift右键无法像1.12.2一样装罐所以在1.19.2中的装罐器调整好UI后直接把水桶放进UI，一两秒后直接游戏崩溃，下面是崩溃报告，没有error，直接崩溃
[crash-2023-09-23_00.06.30-server.txt](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/12705169/crash-2023-09-23_00.06.30-server.txt)
于是我开了新存档，用水单元做了和水桶一样的操作，结果也一样，下面是崩溃报告
[crash-2023-09-23_09.47.20-server.txt](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/12705168/crash-2023-09-23_09.47.20-server.txt)

还有小问题就是金属成型机无法合成，流体弹出升级没有功能
还有希望乾大解决以上问题

**Comment by goingupbao** [comment]
[minecraft-exported-crash-info-2023-09-23T23-15-25.zip](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/12706754/minecraft-exported-crash-info-2023-09-23T23-15-25.zip)
我也是放了个水桶在装罐机里，现在无法进入游戏了
